### PR TITLE
#40 Clickable phone number links, Readme file syntax error

### PR DESCRIPTION
**Pull request by kokigit**
telephone numbers in home.blade , emergency.blade have changed to clickable links and fixed readme file syntax error 'chmod 777 -R storage/'  => 'chmod -R 777 storage/'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,15 +30,15 @@ docker-compose up -d
 docker exec -it reliefsupports_php_1 bash
 composer update
 
-chmod 777 -R storage/
+chmod -R 777 storage/
```
Comment (gayanhewa) [inline README.md:33]: This is alright for now , but you can change this to run ie. `chown nginx:nginx storage` @on the web servers username and group and storage access won't be a problem

**Review by gayanhewa — APPROVED** [review]

File: resources/views/frontend/emergency.blade.php [hunk 2]
```diff
@@ -51,14 +51,14 @@
...
-                <p>කොළඹ : 011  2369134  / 0773184910</p>
-                <p>ගමපහ : 033 22 22235  / 0773273507</p>
-                <p>කෑගල්ල : 035 22 22235  / 0773633293</p>
+                <p>රත්නපුර : <a href="tel:number">045 22 22235</a> / <a href="tel:number">0714423760</a></p>
+                <p>කලුතර : <a href="tel:number">034 22 22235 </a>/ <a href="tel:number">0716814813</a></p>
+                <p>ගාල්ල : <a href="tel:number">091 22 34235</a> / <a href="tel:number">0714415377</a></p>
+                <p>මාතර : <a href="tel:number">041 22 22235</a> / <a href="tel:number">0776864397</a></p>
+                <p>හම්බන්තොට :  <a href="tel:number">047 22 56235</a> / <a href="tel:number">0714441612</a></p>
+                <p>කොළඹ : <a href="tel:number">011  2369134</a>  / <a href="tel:number">0773184910</a></p>
+                <p>ගමපහ : <a href="tel:number">033 22 22235</a>  / <a href="tel:number">0773273507</a></p>
```
Comment (gayanhewa) [inline resources/views/frontend/emergency.blade.php:60]: shouldn't the numbers syntax be `<a href="tel:0332222235">033 22 22235</a>`

**Comment by thilinag** [comment]
Yes. Need to fix before merge

File: resources/views/frontend/emergency.blade.php (hunk 2 above)
Comment (gayanhewa) [reply resources/views/frontend/emergency.blade.php:60]: applepay

**Review by gayanhewa — CHANGES_REQUESTED** [review]
I think the clickable number syntax needs to change for it to work across devices

**Comment by thilinag** [comment]
It will be better to remove all spaces and non numeric characters in href
attribute

**Review by danishka — APPROVED** [review]

**Pull request merged**